add.php

```diff
@@ -114,13 +117,10 @@ function submit() {
             var enterer   = $("#enterer-input").val();
 
             var committees_total = document.getElementsByClassName("committees-checkbox");
-            var committee = "";
-            for(var i = 0; i < committees_total.length; ++i) {
-                if(committees_total[i].checked) {
-                    committee = committee.concat(", "  + committees_total[i].value);
-                }
+            var committee = committees_total.filter(committee => committee.checked).join(", ");
```

Comment:
Only reason I didn't do this last time is compatibility, how far back to we want to support non-ES6 (pre-2015) browsers?

Reply:
Nah I don't think we'll have anyone using old browsers

Reply:
Cool, I have one change to make but it looks good.